colabs/Dane_z_dodatkowymi_kolumnami.ipynb

```diff
@@ -131,13 +131,40 @@
       "cell_type": "code",
       "source": [
         "%%time\n",
-        "data.write.parquet(DATA_WITH_ADDITIONAL_COLUMNS)"
+        "data_splited = data.randomSplit([0.7, 0.1, 0.2], seed=1)"
       ],
       "metadata": {
         "id": "COSijsIZfAcN"
       },
       "execution_count": null,
       "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "train_data = data_splited[0]\n",
+        "validate_data = data_splited[1]\n",
+        "test_data = data_splited[2]"
+      ],
+      "metadata": {
+        "id": "7PeglY6PDd8H"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "%%time\n",
+        "data.write.parquet(TRAIN_DATA)\n",
+        "data.write.parquet(VALIDATE_DATA)\n",
+        "data.write.parquet(TEST_DATA)"
```

Comment:
Rozbiłbym to na 3 osobne komórki i każdą puściłm z `%%time`

Reply:
zmienione